### DISCUSSION TITLE
My reconstruction results don't look normal.

### DISCUSSION THREAD

**Discussion by wwwaaa32498** [astra-toolbox/astra-toolbox#498] [Q&A]
I'm using the python version of astra-toolbox, but there seems to be some problems with the reconstruction. Is it an artefact please?The reconstructed slice is shown below.
![图片2](https://github.com/user-attachments/assets/ca32c69a-3a9a-4259-911f-b4097bebd2f6)

The raw data projection plot looks normal, as shown below. Is this becauseMetal artifact?
![图片1](https://github.com/user-attachments/assets/d4cd4810-22da-4019-8edb-209503ff66f5)

I would be grateful for any help you can give me.

**Comment by askorikov** [comment]
Yes indeed, this looks like severe metal artifacts to me.
